cypress/integration/smoke.spec.ts

```diff
@@ -58,25 +58,25 @@ const addTablePanel = (q: string) => {
   };
 
   e2e.flows.addPanel({
-    matchScreenshot: true,
-    visualizationName: e2e.flows.VISUALIZATION_TABLE,
+    matchScreenshot: false,
     queriesForm: () => {
       fillQuery(q);
       // Blur the editor to execute the query and wait
       cy.get('.panel-content').last().click();
       cy.get('.panel-loading');
       cy.get('.panel-loading', { timeout: 10000 }).should('not.exist');
+      cy.contains('Data does not have a time field').should('exist');
```

Comment:
moving away from screenshots for when we start testing against multiple versions?